lib/index.js

```diff
@@ -52,16 +53,16 @@ module.exports = (referenceDir, config, { watch, fingerprint, sourcemaps, compac
 		buckets[bucket].push(build);
 	});
 
-	let buildAll = files => {
+	let buildAll = new SerializedRunner(files => {
 		return buildStep(buckets.static)(files).
 			then(buildStep(buckets.scripts.concat(buckets.styles))).
 			then(buildStep(buckets.markup));
-	};
-	buildAll();
+	});
+	buildAll.run();
 
 	if(watch) {
 		makeWatcher(config.watchDirs, referenceDir, assetManager.resolvePath).
-			on("edit", buildAll);
+			on("edit", buildAll.rerun);
```

Comment:
This method isn't bound, so I'm _assuming_ that fails silently (i.e. compilation isn't actually being serialized).